src/core_plugins/kibana/public/dashboard/__tests__/dashboard_state.js

```diff
@@ -1,102 +0,0 @@
-import ngMock from 'ng_mock';
```

Comment:
Migrated these to jest tests and modified the index pattern ones to handle the new communication of index pattern from embeddables.